src/Avatar/Avatar.js

```diff
@@ -32,14 +32,9 @@ function getStyles(props, context) {
     },
   };
 
-  if (src && avatar.borderColor) {
+  if (src) {
     Object.assign(styles.root, {
-      background: `url(${src})`,
-      backgroundSize: size,
-      backgroundOrigin: 'border-box',
-      border: `solid 1px ${avatar.borderColor}`,
-      height: size - 2,
-      width: size - 2,
+      boxShadow: `inset 0px 0px 0px 1px ${avatar.borderColor}`,
```

Comment:
Since it's just a single property, we may as well assign it directly instead of using `Object.assign()`